cypress-io/cypress
# fix: make sure to go back to no-specs when delete spec file

Closes #16553


### User facing changelog

In CT when deleting/renaming the spec file we are on, instead of erroring, we roll back to the welcome page. 

- [x] Has the original issue or this PR been tagged with a release in ZenHub? 

### To test

run 

```sh
yarn workspace @packages/runner-ct cypress:open
touch packages/runner-ct/cypress/components/test.spec.tsx
```

- then click on the new test.spec.tsx
- and finally, delete the created file.
- the page should go back to the "no spec selected"
